app/src/main/java/com/sun/basic_japanese/flashcard/eachpage/FlashCardPageFragment.kt

```diff
@@ -35,16 +40,33 @@ class FlashCardPageFragment : Fragment(), FlashCardPageContract.View {
         super.onViewCreated(view, savedInstanceState)
         initPresenter()
         showFlashCard()
+        setEventClick()
+    }
+
+    override fun showItemChanged() {
+        setWordRememberStatus()
     }
 
     private fun initPresenter() {
-        presenter = FlashCardPagePresenter(this)
+        context?.let {
+            val alphabetRepository =
+                AlphabetRepository.getInstance(AlphabetLocalDataSource(AppDatabase.getInstance(it)))
+            flashCardPresenter = FlashCardPagePresenter(this, alphabetRepository)
+        }
     }
 
     private fun showFlashCard() {
         flashCard.apply {
             textWordInternational?.text = alphabet?.romaji
             if (activity != null) displayWordAnimation()
+            setWordRememberStatus()
+        }
+    }
+
+    private fun setWordRememberStatus() {
+        when (flashCard.alphabet?.remember) {
```

Comment:
ông có thể viết như này: 
```suggestion
        switchRemember.isChecked = when (flashCard.alphabet?.remember) {
```